src/core/toga/widgets/multilinetextinput.py

```diff
@@ -19,9 +19,12 @@ class MultilineTextInput(Widget):
     MIN_WIDTH = 100
 
     def __init__(self, id=None, style=None, factory=None,
-                 initial=None, readonly=False, placeholder=None):
+                 initial=None, readonly=False, placeholder=None,
+                 on_change=None):
         super().__init__(id=id, style=style, factory=factory)
 
+        self._on_change = on_change
```

Comment:
Two things are missing here:

1. You should create a `on_change` python [property](https://www.geeksforgeeks.org/python-property-function/) instead of using the private attribute `_on_change`. you can take a look at `toga.TextInput` for inspiration.
2. When setting an `on_change` method, at should call the `set_on_change` method of the implementation level. Again, you can take a look at `toga.TextInput to see how to do it.

Reply:
ok, done